README.md

```diff
@@ -1,102 +1,96 @@
 # remult-better-auth
 
-Adapter to use [better-auth](https://www.better-auth.com) with [remult ORM](https://remult.dev)
+This adapter allows you to use [better-auth](https://www.better-auth.com) with [remult](https://remult.dev).
 
-## Installation
+## Default usage
 
-```console
-
-pnpm i @nerdfolio/remult-better-auth
+```sh
+# Install as dev dependencies.
+npm add better-auth @nerdfolio/remult-better-auth -D
```

Comment:
fine to add `npm` instruction. Let's not remove `pnpm`. I might move this into a pnpm workspace in the near future to make managing them a bit simpler for my workflow.

Reply:
ok 👌